src/components/Footer.js

```diff
@@ -0,0 +1,283 @@
+          Get in touch
+        </h2>
+        <div
+          css={{
+            paddingTop: "30px",
+            listStyleType: "none",
+            ">div": { paddingBottom: "18px" },
+            " a": {
+              textDecoration: "none",
+              textAlign: "left",
+              color: "inherit",
+              fontWeight: "inherit"
+            },
+            display: "grid",
+            [smallerScreens]: {
+              marginLeft: "auto",
+              marginRight: "auto",
+              gridTemplateColumns: "1fr 1fr 1fr 1fr",
+              maxWidth: "196px",
+              paddingTop: "0px",
+              paddingBottom: "60px"
+            }
+          }}
+        >
+          {socialLinks.map(({ icon, label, url }) => (
+            <div
+              key={url}
+              css={{
+                display: "grid",
+                gridTemplateColumns: "min-content auto",
+                ":hover": {
+                  " a": {
+                    color: "#ed253a",
+                    ">div": {
+                      backgroundColor: "#ed253a"
+                    }
+                  }
+                },
+                [smallerScreens]: {
+                  display: "block",
+                  width: "40px",
+                  height: "40px",
+                  marginLeft: "auto",
+                  marginRight: "auto"
+                }
+              }}
+            >
+              <a href={url} css={{}}>
+                <div
+                  css={{
+                    backgroundColor: "#e3e3e3",
+                    mask: `url(${icon}) no-repeat center center`,
+                    maskSize: "100% 100%",
+                    width: "28px",
+                    height: "28px",
+                    marginRight: "16px",
+                    marginTop: "-5px",
+                    ":hover": {
+                      backgroundColor: "#ed253a"
+                    },
+                    [smallerScreens]: {
+                      width: "40px",
+                      height: "40px"
+                    }
+                  }}
+                />
+              </a>
+              <a css={{ [smallerScreens]: { display: "none" } }} href={url}>
+                {label}
+              </a>
+            </div>
+          ))}
+
+          <p
+            css={{
+              ...finePrint,
+              paddingBottom: "32px",
+              paddingTop: "132px",
+              textAlign: "right",
+              [smallerScreens]: { display: "none" }
+            }}
+          >
+            Copyright © 2018 QHacks
+          </p>
+        </div>
+      </div>
+      <div css={{ [largerScreens]: { display: "none" } }}>
+        <p css={{ ...finePrint, paddingBottom: "6px" }}>
+          Queen&#39;s University | 99 University Ave, Kingston, ON
+        </p>
+        <p css={{ ...finePrint, paddingBottom: "6px" }}>
+          {"< />"} with ♡ by Queen&#39;s Students
```

Comment:
You matched the mocks, but i feel like a solid heart would look better (I know, totally minor :slightly_smiling_face:  )

```suggestion
          {"< / >"} with 🖤 by Queen&#39;s Students
```

Reply:
^ nice use of the new GH feature 🌟

Reply:
I'm impartial @RobertWSaunders what are your thoughts?

Reply:
Nice, I wonder if you did it for like 20 things, would it be 20 separate commits, hmmm.

Reply:
Seems like it is a separate commit

Reply:
Will make this change tho!

Reply:
UPDATE: the heart shows up as black, so back to the original it is

Reply:
Sounds good, marking this as resolved!